index.html

```diff
@@ -18,65 +18,89 @@
+                    <a title="The link will take you to my twitter profile" href="https://github.com/MPDADDY">
+                        <i class="fa fa-twitter social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my instagram profile"
+                        href="https://www.instagram.com/mpdaddympd/">
+                        <i class="fa fa-instagram social"></i>
+                    </a>
+                </li>
+                <li>
+                    <a title="The link will take you to my linkin profile"
+                        href="https://www.linkedin.com/in/anold-mwawasi">
+                        <i class="fa fa-linkedin social"></i>
+                    </a>
+                </li>
+            </ul>
         </article>
-        <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work
-            and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
-        </p>
-        <h4>LET'S CONNECT</h4>
-        <ul>
-            <li>
-                <a title="The link will take you to my facebook profile" href="https://www.facebook.com/Mpdaddy/">
-                    <i class="fa fa-facebook social"></i>
-                </a>
-            </li>
-            <li>
-                <a title="The link will take you to my twitter profile" href="https://github.com/MPDADDY">
-                    <i class="fa fa-twitter social"></i>
-                </a>
-            </li>
-            <li>
-                <a title="The link will take you to my instagram profile" href="https://www.instagram.com/mpdaddympd/">
-                    <i class="fa fa-instagram social"></i>
-                </a>
-            </li>
-            <li>
-                <a title="The link will take you to my linkin profile" href="https://www.linkedin.com/in/anold-mwawasi">
-                    <i class="fa fa-linkedin social"></i>
-                </a>
-            </li>
-        </ul>
     </section>
+
     <section class="work_section">
         <article class="sections">
-            <div class="scrnshot">
-                <img src="images/Snapshoot Portfolio (2).png" alt="">
+            <div class="scrnshot width">
+                <img class="mobile" src="images/Snapshoot Portfolio (2).png" alt="">
+                <img class="desktop" src="images/Snapshoot Portfolio - Copy.png" alt="">
             </div>
-            <h2 class="colorh2">Tonic</h2>
-            <div>
+            <div class="width">
+                <h2 class="colorh2">Tonic</h2>
                 <div class="canopy">
                     <h4 class="colorh4">CANOPY</h4>
                     <ul>
-                        <li>
+                        <li class="canopylist">
                             <span>
                                 <img width="8px" height="8px" src="images/Ellipse 12.png" alt="">
```

Comment:
- Just as the previous reviewer mentioned, kindly consider changing the background color of the icons between the list items so it looks similar to that of the Figma design 👍 
- Your Design
<img width="327" alt="Screen Shot 2023-05-12 at 5 59 39 PM" src="https://github.com/MPDADDY/portfolio/assets/102699958/70da0dda-15bb-4383-929e-fdd8064fcd68">

- The Figma Design
<img width="359" alt="Screen Shot 2023-05-12 at 5 59 56 PM" src="https://github.com/MPDADDY/portfolio/assets/102699958/4a3432d8-d837-47c0-a3a1-94cc8b0d628d">